.github/workflows/linux.yml

```diff
@@ -63,7 +64,7 @@ jobs:
         cache-downloads: true
 
     - name: Configure using CMake
-      run: cmake -G Ninja -Bbuild ${{matrix.sys.config-flags}} -DCMAKE_BUILD_TYPE:STRING=${{matrix.config.name}} -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DUSE_DATE_POLYFILL=${{matrix.sys.date-polyfill}} -DBUILD_TESTS=ON -DBUILD_EXAMPLES=ON
+      run: cmake -G Ninja -Bbuild ${{matrix.sys.config-flags}} -DCMAKE_BUILD_TYPE:STRING=${{matrix.config.name}} -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DUSE_DATE_POLYFILL=${{matrix.sys.date-polyfill}} -DBUILD_TESTS=ON -DBUILD_EXAMPLES=ON -DUSE_SANITIZER="address;undefined;leak"
```

Comment:
Note that sanitizers will change the runtime behavior of the program (not it's result) so it's best if it's run as a separate configuration. Maybe add a matrix variables to enable them?

Or do you think it's not worth separating?

Reply:
Idk, if we run it in a separate configuration, we will double all jobs and reach 20 linux jobs.
As you said, the result should not change, so it's not really an issue for me.

Reply:
Ok then lets go with this for now and change later. I suspect it might be problematic to not test exactly the same thing we deliver.